doc/ratbagd.8

```diff
@@ -0,0 +1,19 @@
+.TH ratbagd 8 "Apr 13, 2016" ratbagd
+.SH NAME
+ratbagd \- system daemon to introspect and modify configurable mice
+.SH SYNOPSIS
+.B ratbagd
+.RB [ \-\-verbose ]
+.SH DESCRIPTION
+.B ratbagd
+starts the daemon. It shouldn't be invoked directly;
+.B ratbagd
+is normally started through DBus activation.
```

Comment:
This could be a bit more verbose and mention `ratbagctl` explicitly as command tool.